src/test/java/spoon/test/refactoring/RefactoringTest.java

```diff
@@ -131,4 +132,12 @@ public void testTransformedInstanceofAfterATransformation() {
 		assertEquals("o", instanceofInvocation.getLeftHandOperand().toString());
 		assertEquals("spoon.test.refactoring.testclasses.AClassX", instanceofInvocation.getRightHandOperand().toString());
 	}
-}
+
+	@Test
+	public void testremoveDeprecatedMethods() {
```

Comment:
Would be better to put an ignore on the test, instead of commenting its content, wdyt?